keyboards/kinesis/kint41/keymaps/jtojnar/keymap.c

```diff
@@ -0,0 +1,104 @@
+#include QMK_KEYBOARD_H
```

Comment:
Please include a license header here.

Reply:
1. Since, I am basing this on three other keymaps, should I include their significant contributors in the copyright statement?
2. What should the license be, when none of those had a license header? Could I ever safely assume `GPL-2.0-or-later` for files in the firmware repo even though there is no [DCO or anything](https://wiki.linuxfoundation.org/dco)? Or should I try to contact the original contributors instead?